File: 2138-divide-a-string-into-groups-of-size-k/2138-divide-a-string-into-groups-of-size-k.py
class Solution:
    def divideString(self, s: str, k: int, fill: str) -> List[str]:
        while len(s)%k!=0:
            s +=fill
        res = []
        i = 0
        for i in range(0,len(s),k):
            res.append(s[i:i+k])
        return res
        

File: 2138-divide-a-string-into-groups-of-size-k/2138-divide-a-string-into-groups-of-size-k.cpp
class Solution {
public:
    vector<string> divideString(string s, int k, char fill) {
        while(s.length()%k!=0){
            s += fill ;
        }
        vector<string>res ; 
        int i = 0 ; 
        while(i<s.length()){
            res.push_back(s.substr(i,k)) ; 
            i += k ; 
        }
        return res ; 
        

    }
};